euclidean_gcd/CPP/euclidean_gcd.cpp

```diff
@@ -0,0 +1,24 @@
+#include <bits/stdc++.h>
+#define ll long long
```

Comment:
Use int16/int64/etc, rather than the C type long  [runtime/int] [4]

*Origin: CPPLintBear, Section: `all.cpp`.*